### DISCUSSION TITLE
InPlay soccer score line for in play betting

### DISCUSSION THREAD

**Discussion by footytrade2021** [cloudbet/docs#32] [General]
How can i get the in-play soccer score line together with the odds from the Cloudbet API

**Comment by footytrade2021** [comment]
can i have a response please?

**Comment by footytrade2021** [comment]
@kgravenreuth  @ktalovmorx @lukehoersten @dlackty @joelr

**Comment by kgravenreuth** [comment]
Hi @footytrade2021,

Apologies for the delay in reply. 

As mentioned in [this discussion](https://github.com/Cloudbet/docs/discussions/19), we can provide you information about how your API bets have resulted via the [Bet History](https://www.cloudbet.com/api/?utm_source=github.com&utm_medium=community&utm_campaign=axis&urls.primaryName=Trading#/Trading/BetHistory) endpoint of our Trading API. 

Note that you need to be registered as a Cloudbet player to use the Trading API key with a minimum of 10 euro deposit as mentioned in our [Trading API blog article](https://www.cloudbet.com/en/blog/posts/sports-betting-api-tutorial-with-golang#toc-0?utm_source=github.com&utm_medium=community&utm_campaign=axis)

For example, If you'd like to know the result of a given event, then you can place bets on the Winner or Moneyline markets for that event and then check the bet results to know the event's results.

We are currently not planning to add results or scores to the Feed API, while we thank you for your suggestions above.

Regards,
Klaus Gravenreuth
Product & Engineering @ Cloudbet
